test/test_nn.py

```diff
@@ -7743,6 +7743,47 @@ def test_smoothl1loss_negative_beta_not_supported(self):
         with self.assertRaises(RuntimeError):
             F.smooth_l1_loss(torch.randn(2, 2), torch.randn(2, 2), beta=-1.0)
 
+    def test_smooth_l1_loss_vs_huber_loss(self):
+        def _test_smooth_l1_loss_vs_huber_loss_helper(beta=1.0, require_equal=True):
+            input, target = torch.randn(2, 2), torch.randn(2, 2)
```

Comment:
Add two more test cases:

- contiguous tensors with more than 32 elements in their innermost dimension to trigger CPU vectorization
- discontiguous tensors (transposed and/or every other element)